style/ember/sample.js

```diff
@@ -16,3 +16,13 @@ test("checks the box", function() {
     ok(checkBox.prop("checked"), "box is checked");
   });
 });
+
+import Ember from 'ember';
+import DS from 'ember-data';
+
+const { computed } = Ember;
+const { attr, hasMany, belongsTo } = DS;
+
+export default DS.Model.extend({
+  ...
```

Comment:
That's a good point, I hadn't considered the `Object` scenario.

I think I'd be in favor of doing this whenever it doesn't cause issues but I don't feel strongly either way.